internal/app/wwctl/node/set/root.go

```diff
@@ -74,6 +77,10 @@ func init() {
 	baseCmd.PersistentFlags().BoolVar(&SetNetDevDel, "netdel", false, "Delete the node's network device")
 	baseCmd.PersistentFlags().BoolVar(&SetNetDevDefault, "netdefault", false, "Set this network to be default")
 
+	baseCmd.PersistentFlags().StringVarP(&SetParam, "param", "p", "", "Define custom parameter")
```

Comment:
I'd like to use the option name "key" instead of "param" here (`--key/-k` and `--keydel`). If you concur, will you make that minor mod?

Reply:
sure, no problem.
`param` was based on my limited creativity, I like `key` much better.
I'm also going to change the rest of the code from `Params` to `Keys`, otherwise it's confusing when reading the code.